smart-contract/assembly/contracts/internals/file-list.ts

```diff
@@ -0,0 +1,57 @@
+import { FILES_PATH_LIST } from './const';
+
+/**
+ * Adds a new file path to the list of file paths.
+ * @param filePath - The file path to be added.
+ */
+export function _pushFilePath(filePath: string): void {
+  const filePathList = FILES_PATH_LIST.mustValue();
+  filePathList.push(filePath);
+  FILES_PATH_LIST.set(filePathList);
+}
+
+/**
+ * Removes a file path from the list of file paths.
+ * If the file path is not in the list, this function does nothing.
+ * @param filePath - The file path to be removed.
+ */
+export function _removeFilePath(filePath: string): void {
+  const filePathList = FILES_PATH_LIST.mustValue();
+  if (!_isPathFileInList(filePath)) return;
```

Comment:
is it something expected (that the file to remve does not exists?)

If not its better to just `assert(_isPathFileInList(filePath), "File does not exists")

it may help debuging and avoid hiding unexpected behaviours

Reply:
You're right, done